.github/workflows/examine-core.yml

```diff
@@ -0,0 +1,153 @@
+name: Examine the Rust Core
+on:
+  push:
+    branches:
+      - dev
+    paths:
+      - '.github/workflows/examine-core.yml'
+      - '.github/actions/**'
+      - 'src/**'
+      - 'tests/**'
+      - 'examples/**'
+      - 'Cargo.toml'
+      - 'Cargo.lock'
+  pull_request:
+    branches:
+      - dev
+    paths:
+      - '.github/workflows/examine-core.yml'
+      - '.github/actions/**'
+      - 'src/**'
+      - 'tests/**'
+      - 'examples/**'
+      - 'Cargo.toml'
+      - 'Cargo.lock'
+  schedule:
+    - cron: '0 1 * * *'
```

Comment:
With that it will run automatically every day, additionally to the pull requests to the dev branch?

Reply:
Additionally every job has an `if` statement constraining when they'll be run. For example, none but `audit` will be run every day because all other jobs are marked as `if: ${{ ! github.event.schedule }}`.

This is actually a trick to merge the auditing workflow with the crate-wise workflows. `audit.yml` used to be a separate workflow triggered every day. Now, all crate-wise workflows are triggered every day, but when the workflows are run because of the `schedule` event, only the `audit` job will be run. This reduces redundancy.